examples/examples_nodejs_test.go

```diff
@@ -201,6 +202,24 @@ func TestVpcMultipleSimilarSubnetSpecArgs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccEcsParallel(t *testing.T) {
+	maxDuration(15*time.Minute, t, func(t *testing.T) {
```

Comment:
Deploying and updating 10 of these ECS services took 1+ hours before. Now that it's all parallel it runs in under 10min